modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/application/mgt/DefaultAuthSeqManagementTestCase.java

```diff
@@ -0,0 +1,343 @@
+import org.wso2.carbon.identity.application.common.model.xsd.AuthenticationStep;
+import org.wso2.carbon.identity.application.common.model.xsd.DefaultAuthenticationSequence;
+import org.wso2.carbon.identity.application.common.model.xsd.LocalAndOutboundAuthenticationConfig;
+import org.wso2.carbon.identity.application.common.model.xsd.LocalAuthenticatorConfig;
+import org.wso2.carbon.identity.application.common.model.xsd.ServiceProvider;
+import org.wso2.carbon.identity.application.mgt.defaultsequence.stub.IdentityDefaultSeqManagementServiceDefaultAuthSeqMgtException;
+import org.wso2.identity.integration.common.clients.application.mgt.ApplicationManagementServiceClient;
+import org.wso2.identity.integration.common.clients.application.mgt.DefaultAuthSeqMgtServiceClient;
+import org.wso2.identity.integration.common.utils.ISIntegrationTest;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.Scanner;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertNotNull;
+import static org.testng.Assert.assertNull;
+import static org.testng.Assert.fail;
+
+public class DefaultAuthSeqManagementTestCase extends ISIntegrationTest {
+
+    private DefaultAuthSeqMgtServiceClient defaultAuthSeqMgtServiceClient;
+    private ApplicationManagementServiceClient applicationManagementServiceClient;
+    private final String applicationName1 = "TestServiceProvider1";
+    private final String seqName = "default_sequence";
+    private final String seqDesc1 = "This is a seq with basic and totp with role based adaptive authentication";
+    private final String seqDesc1_updated = "This is a seq with basic and totp with role based adaptive authentication" +
+            " for admin role";
+    private final String seqDesc1_updated2 = "This is a seq with basic authentication";
+    private final String seqFileName1 = "valid_default_seq";
+    private final String seqFileName2 = "invalid_content_default_seq";
+    private final String seqFileName3 = "incorrect_default_seq";
+    private final String seqFileName4 = "invalid_xml_element_default_seq";
+    private String seqContent1;
+    private String seqContent2;
+    private String seqContent3;
+    private String seqContent4;
+    private final String applicationName = "TestServiceProvider";
+
+    @BeforeClass(alwaysRun = true)
+    public void testInit() throws Exception {
+
+        super.init();
+        ConfigurationContext configContext = ConfigurationContextFactory.createConfigurationContextFromFileSystem(null
+                , null);
+        defaultAuthSeqMgtServiceClient = new DefaultAuthSeqMgtServiceClient(sessionCookie, backendURL, configContext);
+        applicationManagementServiceClient = new ApplicationManagementServiceClient(sessionCookie, backendURL,
+                configContext);
+        seqContent1 = readResource("defaultAuthSeq/" + seqFileName1 + ".xml");
```

Comment:
Why can't we assign private final String seqFileName1 = "valid_default_seq.xml"; directly, without later appending ".xml"?